environment.yaml

```diff
@@ -1,5 +1,6 @@
-name: base
+name: qa
```

Comment:
Should probably keep the name unchanged for now. I had issues today with getting a separate gpu image to build on base and had to change, did you have similar trouble? I could see the default docker installing everything in a separate environment and assuming `run.sh` and `cli` run `source activate qa`